### ISSUE TITLE
Navigation - The NEXT button in the SvelteKit documentation malfunctions

### ISSUE THREAD

**Issue by lukaszpolowczyk** [sveltejs/kit#4581]
### Describe the bug

Navigation - The NEXT button in the SvelteKit documentation malfunctions.

The https://kit.svelte.dev/docs/service-workers url looks like this - wrong title, wrong NEXT button, wrong PREVIOUS button.
Clicking the back/forward button, NEXT/PREVIOUS, the link in the side panel - none of them set the title or names of the NEXT and PREVIOUS buttons. The links work, but the names of NEXT and PREVIOUS and title are wrong.
The tab title changes.
![Screenshot_2022-04-09 SvelteKit docs(2)](https://user-images.githubusercontent.com/16800535/162552031-e632eaf1-67a5-4ba4-9303-6b258b8be5a8.png)

### Reproduction

Difficult to reproduce.
I was just going through one by one, from https://kit.svelte.dev/docs/introduction to https://kit.svelte.dev/docs/service-workers and then I noticed it.

But when I do it again now in a new tab, the error is gone.

This has happened a few times already, right after the introduction of the search engine and subdivision it happened.

### System Info

```shell
System:
    OS: Linux 5.10 Arch Linux
    CPU: (4) x64 AMD Athlon(tm) X4 950 Quad Core Processor
    Memory: 3.03 GB / 15.07 GB
    Container: Yes
    Shell: 5.1.8 - /bin/bash
  Binaries:
    Node: 16.3.0 - /usr/bin/node
    Yarn: 1.22.10 - /usr/bin/yarn
    npm: 7.17.0 - /usr/bin/npm
  Browsers:
    Brave Browser: 100.1.37.109
    Firefox: 87.0
```

### Severity

annoyance

**Comment by Mlocik97** [comment]
tested in Firefox, can't reproduce. Try to clear cache in browser.

**Comment by bluwy** [comment]
No issues for me too.

**Comment by lukaszpolowczyk** [comment]
This is really hard to reproduce, but it is happening with a simple review of the documentation. This error has come up several times over the course of several weeks.
MAYBE it would be easier to look at the code and then determine what could be the cause?
If not, then this issue needs to be closed for now. :/

**Comment by Rich-Harris** [comment]
I removed the `documentation` label because if this is a bug, it's a bug in SvelteKit, and this is just a particular manifestation of it. It really does need a repro though

**Comment by Rich-Harris** [comment]
Are you seeing any errors in your console when this happens?

**Comment by lukaszpolowczyk** [comment]
@Rich-Harris No, there are no errors in the console.

**Comment by lukaszpolowczyk** [comment]
@Rich-Harris OMG. I looked at my Firefox extensions, and realized I use this one sometimes:
https://addons.mozilla.org/firefox/addon/traduzir-paginas-web/

**It just swaps the text on the page, to the translated version...**
So it's not surprising that some prcise connection between svelte and TextNode gets broken.
:roll_eyes: 

The question is, is it possible and does it even make sense to include this? Probably not?
I can only apologize. I feel silly now. :pray:

**Comment by Mlocik97** [comment]
it's impossible to fix website so it works correctly with all possible browser extensions... correct solution is that user disable extension on specific website.

**Comment by bluwy** [comment]
Closing as it's caused by a browser extension

**Comment by bluwy** [comment]
I didn't notice this was caused by the built-in translator in Chrome too. Perhaps there's more to look into in that case. Re-opening.

**Comment by dummdidumm** [comment]
To highlight (because I didn't see it at a glance previously): This is reproducible by using Chrome's builtin Google Translate on the docs. The headline and the previous/next buttons do not update their text.

My assumption is that this is a Svelte core issue. Google Translate is messing with the text content, so it gets out of sync with Svelte, which no longer is able to upgrade the text nodes (they are different nodes now).

**Comment by lukaszpolowczyk** [comment]
@dummdidumm Yes, it's a SvelteJS problem. I mistakenly thought it was a SvelteKit navigation problem, that's why I started the thread in the SvelteKit repo.
It can be moved or something.
